.gitmodules

```diff
@@ -7,3 +7,7 @@
 [submodule "src/kobocat"]
 	path = src/kobocat
 	url = https://github.com/kobotoolbox/kobocat.git
+[submodule "src/kobocat-template"]
```

Comment:
What repo is this @boney-bun ?

Reply:
it's a new repo i've created for the kobocat's style.
originally, the style is generated from https://github.com/kobotoolbox/kobocat-template. 
we need to override the repo, so i added the repo.